network/network-address/src/lib.rs

```diff
@@ -321,6 +321,45 @@ impl NetworkAddress {
             .push(Protocol::Handshake(handshake_version))
     }
 
+    /// Check that a `NetworkAddress` looks like a typical LibraNet address with
+    /// associated protocols.
```

Comment:
some comments about what is a typical address?

Reply:
I would suggest renaming this to "canonical". We are defining what a proper libranet address is. (We should probably give a name to other types of address as well.)